Xray absorption correction Algorithm (#30089)
mantidproject/mantid: docs/source/algorithms/XrayAbsorptionCorrection-v1.rst

.. algorithm::

.. summary::

.. relatedalgorithms::

.. properties::

Description
-----------

This algorithm calculates the correction factors due to the absorption of Xrays 

> by the sample and applies it to Input Workspace. This is done by determining path of an xray from a 

Reviewer: Does the algorithm apply the correction to the input workspace? It looks to me as though the algorithm calculates the correction factor and then (in line with the other absorption correction algorithms) the user is responsible for applying the correction afterwards - this is normally done by dividing a real "signal" workspace by the calculated correction factor.

Author: It does not apply the correction it would have to be done by the user

Reviewer: The reason I asked is that the first sentence I've attached my comment to says "...and applies it to the Input Workspace". So I think this text needs to be removed?

.. algorithm::

.. summary::

.. relatedalgorithms::

.. properties::

Description
-----------

This algorithm calculates the correction factors due to the absorption of Xrays 

> by sample and applies it to Input Workspace. This is done by determining path of xray from

Reviewer 2: ```suggestion
by the sample and applies it to Input Workspace. This is done by determining path of an xray from a
```

> muon in sample to detector and using the calculated distance in sample to calculate 

Reviewer 2: I'm not sure what distance it is using. You could put:

muon in the sample to the detector. It calculates the distance between ???? and the sample to calculate the correction function using

then you should use the maths formatting for the equation

correction factor using exp(-absorptioncoefficient(energy) * distance).

Input Workspace Requirements
############################

The algorithm will compute the correction factors on a bin-by-bin basis for each spectrum within

> the input workspace. The following assumptions on the input workspace will are made:

Reviewer 2: ```suggestion
the input workspace. The following assumptions on the input workspace are made:
```

   
     - properties of the sample and optionally its environment have been set with :ref:`SetSample <algm-SetSample>`
	 

>      - Xray Attenuation profile data must be provided using :ref:`SetSampleMaterial <algm-SetSampleMaterial>`

Reviewer 2: ```suggestion
     - Xray Attenuation profile data is provided by using :ref:`SetSampleMaterial <algm-SetSampleMaterial>`
```

>      - Muon Implantation Profile must have only one spectrum and muon depth should be x data and intensity should be y data.

Reviewer 2: ```suggestion
     - Muon Implantation Profile has a single spectrum. The x data is the muon depth (in ???) and the y data is intensity (in ???).
```
```suggestion
     - Muon Implantation Profile must have only one spectrum and muon depth should be x data and intensity should be y data.
```

> Input Parameters

Reviewer 2: dont need this section, it is auto generated from the algorithm properties
